external-crates/move/crates/move-vm-integration-tests/src/compiler.rs

```diff
@@ -58,3 +58,7 @@ pub fn compile_modules(s: &str) -> Result<Vec<CompiledModule>> {
 pub fn as_module(unit: AnnotatedCompiledUnit) -> CompiledModule {
     unit.named_module.module
 }
+
+pub fn serialize_module(module: &CompiledModule, binary: &mut Vec<u8>) -> Result<()> {
```

Comment:
nit, maybe a slightly different name. I was confused at first why this existed. Maybe a comment? idk